src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.h

```diff
@@ -149,6 +149,11 @@ namespace xdp {
       EventVector memoryCounterResetEvents;
```

Comment:
Copyright date needs to be updated on line 2.

Reply:
Done